src/utils/getRoam.ts

```diff
@@ -32,7 +32,7 @@ type GetRoamResult =
  *
  * Connectivity Monitoring (4) object does not support timestamp, for that reason timestamp value is taken from
  * Device (3) object.
- * @see {@link ../../adr/010-roam-timestamp-not-supported-by-lwm2m.md}
+ * @see {@link ../../adr/005-roam-timestamp-not-supported-by-lwm2m.md}
  */
 export const getRoam = ({
 	connectivityMonitoring,
```

Comment:
**Suggestion:** The function `getRoam` doesn't handle the case when `connectivityMonitoring` or `device` is undefined. This could lead to a runtime error if these parameters are not provided.
```suggestion
export const getRoam = ({
    connectivityMonitoring,
    device,
}: GetRoamParameters): GetRoamResult => {
    if (!connectivityMonitoring || !device) {
        return { error: new Error('Missing required parameters') }
    }
    // ...
}
```